types.d.ts

```diff
@@ -236,7 +229,7 @@ declare namespace PgBoss {
     id: string,
     name: string,
     options: WorkOptions,
-    state: 'created' | 'retry' | 'active' | 'completed' | 'expired' | 'cancelled' | 'failed',
+    state: 'created' | 'retry' | 'active' | 'completed' | 'cancelled' | 'failed',
```

Comment:
Could you add "stopping" here? Then we can complete https://github.com/timgit/pg-boss/issues/422 as well.